### PR TITLE
[DT-132] Include workflow event with error in strongly typed error message

### PR DESCRIPTION
**Pull request by laurakwhit**
## What was changed

When a strongly-typed error occurs with an event in the workflow history the expandable event is included in the `Alert` along with the error message.

## Why?

Users can see event details immediately.

## Checklist

1. Closes: `DT-132` 

2. How was this tested:

- [ ] `getStartedCompletedAndTaskFailedEvents.test.ts`
- [ ]  Run `main.go` code attached to task for a `NonDeterministicError`
3. Any docs updates needed? n/a

### CODE REVIEW

File: src/lib/components/event/event-summary-table.svelte [hunk 1]
```diff
@@ -54,7 +62,11 @@
       </div>
     </div>
   </div>
-  <div class="table-header-row-responsive rounded-t-md">
+  <div
+    class="table-header-row-responsive rounded-t-md {typedError
+      ? 'header-hidden-responsive'
+      : ''}"
```
Comment (rossedfort) [inline src/lib/components/event/event-summary-table.svelte:68]:
```suggestion
    class="table-header-row-responsive rounded-t-md"
    class:header-hidden-responsive={typedError}
```

Comment (rossedfort) [reply src/lib/components/event/event-summary-table.svelte:68]: Actually, I think you could just do `class:hidden={typedError}` for this one.

**Review by rossedfort — COMMENTED** [review]
Could just use some css simplification but looks good otherwise!

File: src/lib/components/event/event-summary-table.svelte [hunk 2]
```diff
@@ -25,9 +26,16 @@
   }
 </script>
 
-<section class="event-table" data-cy="event-summary-table">
+<section
+  class="event-table {typedError
+    ? 'table border border-yellow-700'
+    : 'border-2 border-gray-900 xl:table'}"
```
[the remark this answers is not in the document]
Comment (rossedfort) [reply src/lib/components/event/event-summary-table.svelte:32]: Or at least the `table border border-yellow-700` styles extracted into a `typed-error` class that you add based on `typedError` like you did in [src/lib/components/event/event-summary-row.svelte](https://github.com/temporalio/ui/pull/927/files/39de2f86a53121be814fee8fbcb777c36cbd51b3#diff-3b3a513166ebdf3c4dfc0225f1eb9a20c9738a7eee06bf3a0039e7da5d75ddea)

[the remark this answers is not in the document]
Comment (laurakwhit) [reply src/lib/components/event/event-summary-table.svelte]: I've added an `error-table` class that will overwrite the original border properties in `event-table`.

File: src/lib/components/event/event-summary-table.svelte (hunk 1 above)
Comment (laurakwhit) [reply src/lib/components/event/event-summary-table.svelte]: `class:hidden={typedError}` didn't work 😞 But updated to your first suggestion, thank you!

**Review by rossedfort — APPROVED** [review]

**Pull request merged**